.circleci/config.yml

```diff
@@ -8,31 +8,37 @@ executors:
 
 # reusable command for all jobs
 commands:
-  checkout_from_cache:
-    description: 'To checkout and restore the dependencies cache'
+  checkout_from_workspace:
+    description: 'To checkout and attach the workspace'
     steps:
       - checkout
-      - restore_cache:
-          keys:
-            - dependency-cache-{{ checksum "package.json" }}
-            - dependency-cache-
+      - attach_workspace:
+          at: ~/webex-components
 
 # define jobs
 jobs:
   install:
     executor: main-executor
     steps:
-      - checkout_from_cache
+      - checkout
+      - restore_cache:
+          keys:
+            - dependency-cache-{{ checksum "package-lock.json" }}
+            - dependency-cache-
       - run:
           name: Install Dependencies
           command: npm ci
       - save_cache:
-          key: dependency-cache-{{ checksum "package.json" }}
+          key: dependency-cache-{{ checksum "package-lock.json" }}
```

Comment:
decided to pick `package-lock` instead of `package`, since we are using `npm ci`